src/resources/questions-ingredients/question-1/view.html

```diff
@@ -0,0 +1,3 @@
+<small class="hint"
```

Comment:
@DanielTurner  why is this not inside the form.io page?

Reply:
It is inside the formio page, this is the component developed outside the formio environment then built and pasted in. That way we easily get around single user on formio, resolve any issues with formio backups by using a more reliable code repository.